src/transform/mod.rs

```diff
@@ -557,6 +557,7 @@ mod test {
 
   fn roundtrips<T: Pixel>() {
     let combinations = [
+      (TX_4X4, WHT_WHT, 0),
```

Comment:
This replaces `test_lossless_roundtrip()` as it can now be expressed by `test_roundtrip()`.